lib/Conversation/index.js

```diff
@@ -59,34 +59,38 @@ class Conversation extends Component {
       'is-active': showComments
     });
     const { id, comments, ...rest } = this.props;
+    const existingConversation =
+      this.props.resolveConversation && this.props.comments.length > 0;
+    const placeholderText = existingConversation
```

Comment:
I feel like `placeholderText` should just be a prop that is passed in. What if we want to change the text to "Reply to this thread..."?

That way we remove the idea of an "existing" conversation too. Which simplifies the component.